### ISSUE TITLE
Ability to use the --csv option to output the results to specified directory

### ISSUE THREAD

**Issue by DFIRJoe** [countercept/chainsaw#9]
Hi, 
Great Tool! Would love to see the --csv option to have the ability to output/write the csv files to a specified directory similar to the --json option?  With that, it should be pretty easy to integrate with other tools, such as integrating it as a kape module.

**Comment by high101bro** [comment]
Agreed! Awesome tool! I would definetly apprecaite being able to specify the csv save location

**Comment by fscc-jamesd** [comment]
Good idea, I probably should have implemented it in this way from the start to be honest. I'll take a look.

**Comment by rathbuna** [comment]
Seconded. Will update KAPE Module once this is implemented.
